.github/workflows/ci.yml

```diff
@@ -52,11 +52,34 @@ jobs:
     steps:
       - name: Prepare OS
         run: |
-          (apt-get update && apt-get install curl -y) || true
-          yum install tar gzip -y || true
+          (apt-get update && apt-get install file curl -y) || true
+          yum install file tar gzip -y || true
       - uses: actions/checkout@v3
       - name: Run installer.sh
         run: sh docs/.vuepress/public/installer.sh
+      - name: Check kumactl architecture
+        run: file ./kuma-*/bin/kumactl | grep x86-64 &> /dev/null
+  installer-sh-arm64:
```

Comment:
Can't architecture be another dimension in the matrix? We don't have to repeat the `apt-get install` code and it's consistent.

Reply:
Steps are using different actions. The action used for `arm64` can't be used for `amd64` because of the lack of images supported above. As far as I know, we don't support other architectures so I am not sure if it's worth it.

Reply:
Agreed, I think it'd make the job more complex to set up the matrix properly and handle selectively running `run-on-arch-action`